ui/lib/replication/addon/routes/application.js

```diff
@@ -16,8 +16,7 @@ export default Route.extend(ClusterRoute, {
   },
 
   model() {
-    const activeClusterId = this.auth.activeCluster;
-    return this.store.peekRecord('cluster', activeClusterId);
```

Comment:
I like this refactor you did! Adding it to the service instead of peeking everytime.